api/src/routing.js

```diff
@@ -441,7 +440,7 @@ app.postJson('/api/v1/places/:id', function(req, res) {
 
 app.postJson('/api/v1/people', function(req, res) {
   auth
-    .check(req, 'can_create_people')
+    .check(req, ['can_edit', 'can_create_people'])
```

Comment:
Change to `hasAllPermissions` (as above).

Reply:
`check` was refactored